src/test/regress/expected/relation_access_tracking.out

```diff
@@ -577,20 +593,20 @@ BEGIN;
 (1 row)
 
 	SELECT * FROM relation_acesses  WHERE table_name IN ('table_6', 'table_1') ORDER BY 1,2;
- table_name |  select_access  |  dml_access  |  ddl_access  
-------------+-----------------+--------------+--------------
- table_1    | parallel_access | not_accessed | not_accessed
- table_6    | parallel_access | not_accessed | not_accessed
+ table_name |  select_access  |      dml_access       |      ddl_access       
+------------+-----------------+-----------------------+-----------------------
+ table_1    | parallel_access | not_parallel_accessed | not_parallel_accessed
+ table_6    | parallel_access | not_accessed          | not_accessed
```

Comment:
`table_6` is a reference table. Why is it marked as `parallel_access` and not `reference_table_access`?

Reply:
I also realized this. 

The query is reference table join with a distributed table. That seems like a minor bug in the implementation before this change. We could fix it, but seems not appropriate for backport?